### PR TITLE
delete: The process cannot access the file because it is being used b…

### PR DESCRIPTION
**Pull request by RA489**
The process cannot access the file because it is being used by another process.
fixes https://github.com/kubernetes/minikube/issues/6804

### CODE REVIEW

**Comment by medyagh** [comment]
@RA489 do you mind repllicating this error if you have access to windows machine and putting the PR description the output to see if the regex matches ? I suspec this regex might not be accurate.

**Comment by tstromberg** [comment]
I can confirm the regexp:

`X Unable to remove machine directory: %v: remove C:\Users\spam1\.minikube\machines\minikube\minikube\Snapshots\D9776725-C283-4806-9F5A-D484B8E3496E.vmcx: The process cannot access the file because it is being used by another process.*`

If you don't mind resolving conflicts, I think this can be merged.

**Comment by RA489** [comment]
@tstromberg Merge conflict resolved now.

File: pkg/minikube/problem/err_map.go [hunk 1]
```diff
@@ -109,6 +109,12 @@ var vmProblems = map[string]match{
 		Issues: []int{4511},
 		GOOS:   []string{"windows"},
 	},
+  
+	"HYPERV_FILE_ACCESS_DENIED": {
+		Regexp: re(`The process cannot access the file because it is being used by another process`),
+		Advice: "Hyper-V Manager locked the file. Please stop the Hyper-V Manager, and run 'minikube delete'.",
+		URL:    "https://docs.docker.com/machine/drivers/hyper-v/",
+
```
Comment (tstromberg) [inline pkg/minikube/problem/err_map.go:117]: `[/home/travis/gopath/src/github.com/kubernetes/minikube/pkg/minikube/problem/err_map.go:323:2: missing ',' before newline in composite literal /home/travis/gopath/src/github.com/kubernetes/minikube/pkg/minikube/problem/err_map.go:326:1: expected `

This is missing a line that says:

```},```

**Comment by tstromberg** [comment]
Closing as obsolete.

**Pull request closed without merging**